lib/my_john_deere_api/net_http_retry/decorator.rb

```diff
@@ -1,30 +1,35 @@
 module MyJohnDeereApi
   module NetHttpRetry
     class Decorator
-      attr_reader :object, :request_methods, :retry_delay_exponent, :max_retries, :response_codes
+      attr_reader :object, :request_methods, :retry_delay_exponent, :max_retries, :retry_codes, :valid_codes
 
       DEFAULTS = {
         request_methods:      [:get, :post, :put, :delete],
         retry_delay_exponent: 2,
         max_retries: 12,
-        response_codes: ['429', '503']
+        retry_codes: ['429', '503'],
```

Comment:
Should we add `409` to this list? Or will that be rejected elsewhere.

Reply:
If we really want, we can handle 409's at the application level (ie, Periodic Pigeon) by ignoring the 409's, but I don't think it's a good idea to monkey with them in this gem. They're legitimate error codes that should bubble up.